rfcs/0000-rfc-template.md

```diff
@@ -0,0 +1,64 @@
+---
+Status: Accepted
+Champions: @justinfagnani
+PR:
+---
+
+# {{ TITLE }}
+
+Brief summary paragraph of the RFC.
+
+## Objective
+
+What is the RFC trying to accomplish?
+
+### Goals
+- Specific goals of the RFC
+
+### Non-Goals
+- Specific non-goals of the RFC
+
+## Motivation
+
+Why is the RFC neccessary? What background information is needed to understand why?
+
+## Detailed Design
+
+Describe the change in detail. Include API changes and usage examples. Add your own sub-headings as necessary.
```

Comment:
Is this intended to be the section where the bulk of the actual description of the new feature is expected to be detailed?

It contains a bunch of subsections that are more meta about the change (number of PRs, backward compat, testing, etc.), so it's maybe a little unclear where the meat of the proposal goes. Perhaps all the subsections under detailed design could go under a separate `## Implementation Considerations` section, so it's more clear that `## Detailed design` is for the meat?

Reply:
SGTM